photon.h

```diff
@@ -48,17 +48,25 @@ const uint64_t INIT_IO_DEFAULT = INIT_IO_LIBCURL;
 
 #undef SHIFT
 
+struct PhotonOptions {
+    int libaio_ctx = 2048;
```

Comment:
uint16_t libaio_ctx = 2048;   // perhaps smaller?

Reply:
Just keep as same as it before.

It could be much smaller as default value, like 256 or event 64.